bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/link/LinkUtil.java

```diff
@@ -267,4 +268,12 @@ private static String replaceEncodedCharactersInFragment(final String str) {
             .replace("%2F", "/")
             .replace("%3F", "?");
     }
+
+    private static boolean isNotMailToLink(String link) {
```

Comment:
It's recommended to use positive naming in such test methods.
Instead of isNotMailToLink() use isMailToLink() and change the code accordingly.